src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -512,7 +512,7 @@ void EnGinkoMan_SetupBankAward(EnGinkoMan* this) {
 
 void EnGinkoMan_BankAward(EnGinkoMan* this, GlobalContext* globalCtx) {
     if (Actor_HasParent(&this->actor, globalCtx)) {
-        // ? when would bank have a parent?
+        // Parent is Link when starting to receive the award
```

Comment:
```suggestion
        // Parent is Player when starting to receive the award
```
Just a styling thing we do in the repo, we try to abstract away from Link and consider just the main Player, ex. Kafei being the main player (applies to the comment below as well)

Reply:
Didn't see the suggested change until after I already changed it to 'the player'. Let me know if that's not okay.